joomla/joomla-cms
# Administrator main menu list improvements

#### Description

This PR does some improvements to administrator main menu list:
- corrects an html error (missing " between href and title)
- correct spaces between table lines -> (p to div)
- put counters items colors like in categories, banners, contacts components
- code style: adds ; and spaces at the end of php code
- some minor adjustments
##### Before patch:

![image](https://cloud.githubusercontent.com/assets/9630530/12080986/3f09f71c-b264-11e5-9250-1769755dfcef.png)
##### After patch:

![image](https://cloud.githubusercontent.com/assets/9630530/12080941/29d62a56-b263-11e5-8380-58c50ab9fa2d.png)
#### How to test
1. Fresh Joomla install with multilanguage (several languages).
2. Apply this patch
3. Go to menus view
4. Check if everything is ok.
